Bottom: da0b296334a36dadc4101fc078c3f1d99cf3655d
Top:    171d259731d98d261a2c94259057a57aeeb69854
Author: Ian Kent <raven@themaw.net>
Date:   2013-05-02 14:51:12 +0800

Refresh of autofs-5.0.7-use-MS_UNBINDABLE-for-autofs-mounts.patch

---

diff --git a/lib/mounts.c b/lib/mounts.c
index 550445c..cfd303c 100644
--- a/lib/mounts.c
+++ b/lib/mounts.c
@@ -102,7 +102,7 @@ unsigned int query_kproto_ver(void)
 		return 0;
 	}
 
-	if (mount("automount", t_dir, "autofs", MS_MGC_VAL, options)) {
+	if (mount("automount", t_dir, "autofs", MS_UNBINDABLE|MS_MGC_VAL, options)) {
 		close(pipefd[0]);
 		close(pipefd[1]);
 		rmdir(t_dir);
